src/App.tsx

```diff
@@ -17,18 +57,38 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setFilterSelect={setFilterSelect}
+                setInputSelect={setInputSelect}
+                inputSelect={inputSelect}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todoList ? (<Loader />) : (
+                <TodoList
```

Comment:
This syntax looks better in my opinion 
```suggestion
              {!todoList ? (
                <Loader />
              ) : (
                <TodoList
```